### DISCUSSION TITLE
What is function component in React?

### DISCUSSION THREAD

**Discussion by 0xdev99** [community/community#20747] [Programming Help] [answered]
What is function component in React?
I can't understand exactly.

**Comment by dreamboat999** [comment, marked as answer]
A React functional component is a simple JavaScript function that accepts props and returns a React element. After the introduction of React Hooks, writing functional components has become the ​standard way of writing React components in modern applications.

**Comment by K-0817** [comment]
Usually, a component provides a particular function or group of related functions. In programming design, a system is divided into components that in turn are made up of modules. Component test means testing all related modules that form a component as a group to make sure they work together.

**Comment by AbrorjonWeb** [comment]
I hate programming 😭
